ci/staging/migrate-database.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+APP_NAME="phoenix-starter"
```

Comment:
Now that my PR is merged, you can `source bin/env.sh`: https://github.com/newaperio/phoenix_starter/blob/main/bin/env.sh